tinygrad/renderer/cstyle.py

```diff
@@ -328,7 +328,8 @@ class WGSLLanguage(CStyleLanguage):
   barrier="workgroupBarrier();"
   generic_var_prefix = "var "
   external_local_bufs = True
-  code_for_op = { **CStyleLanguage().code_for_op, BinaryOps.CMPLT: lambda x,y,dtype: f"f32({x}<{y})", TernaryOps.MULACC: lambda x,y,z,dtype: f"fma({x},{y},{z})", TernaryOps.WHERE: lambda a,b,c,dtype: f"select({c},{b},{a}!=0.)" }
+  code_for_op = { **CStyleLanguage().code_for_op, **{opcode: lambda a,b,dtype,op=op: f"bool(i32({a}){op}i32({b}))" if dtype == dtypes.bool else f"({a}{op}{b})" for opcode,op in [(BinaryOps.MUL,"*"),(BinaryOps.ADD,"+"),(BinaryOps.SUB,"-"),(BinaryOps.DIV,"/")]},
```

Comment:
This is needed to convert `bool` binops to `int`, since you can't do `bool` binops in `wgsl`.